### DISCUSSION TITLE
v1.1.0.rc1

### DISCUSSION THREAD

**Discussion by sdebruyn** [dbt-msft/dbt-sqlserver#234] [General]
### Summary

Changeling can be found [here](https://github.com/dbt-msft/dbt-sqlserver/blob/master/CHANGELOG.md)

#### Features

* update to dbt 1.1

#### Fixes

* [#194](https://github.com/dbt-msft/dbt-sqlserver/pull/194) uppercased information_schema ([@TrololoLi](https://github.com/TrololoLi))
* [#215](https://github.com/dbt-msft/dbt-sqlserver/pull/215) Escape schema names so they can contain strange characters ([@johnf](https://github.com/johnf))

#### Chores

* Documentation on how to contribute to the adapter
* Automatic release process by adding a new tag
* Consistent code style with pre-commit
* [#201](https://github.com/dbt-msft/dbt-sqlserver/pull/201) use new dbt 1.0 logger ([@semcha](https://github.com/semcha))
* [#216](https://github.com/dbt-msft/dbt-sqlserver/pull/216) use new dbt testing framework ([@dataders](https://github.com/dataders) & [@sdebruyn](https://github.com/sdebruyn))

## Commits & PRs

* move to 1.0 by @dataders in https://github.com/dbt-msft/dbt-sqlserver/pull/181
* set env vars explicitly to fix circle ci check by @sdebruyn in https://github.com/dbt-msft/dbt-sqlserver/pull/198
* Pre-commit setup by @sdebruyn in https://github.com/dbt-msft/dbt-sqlserver/pull/228
* Fix azsql resume by @sdebruyn in https://github.com/dbt-msft/dbt-sqlserver/pull/229
* remove aiohttp dependency by @sdebruyn in https://github.com/dbt-msft/dbt-sqlserver/pull/230
* Bump docker/build-push-action from 2.10.0 to 3.0.0 by @dependabot in https://github.com/dbt-msft/dbt-sqlserver/pull/222
* Bump docker/login-action from 1.14.1 to 2.0.0 by @dependabot in https://github.com/dbt-msft/dbt-sqlserver/pull/221
* Bump docker/metadata-action from 3.7.0 to 4.0.1 by @dependabot in https://github.com/dbt-msft/dbt-sqlserver/pull/220
* Escape schema names so they can contain strange characters by @johnf in https://github.com/dbt-msft/dbt-sqlserver/pull/215
* uppercased information_schema resolves #193 by @TrololoLi in https://github.com/dbt-msft/dbt-sqlserver/pull/194
* New logging by @semcha in https://github.com/dbt-msft/dbt-sqlserver/pull/201
* build one docker img per python version by @sdebruyn in https://github.com/dbt-msft/dbt-sqlserver/pull/231
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/dbt-msft/dbt-sqlserver/pull/232
* switch to new adapter test framework by @dataders in https://github.com/dbt-msft/dbt-sqlserver/pull/216

## New Contributors
* @dependabot made their first contribution in https://github.com/dbt-msft/dbt-sqlserver/pull/222
* @johnf made their first contribution in https://github.com/dbt-msft/dbt-sqlserver/pull/215
* @TrololoLi made their first contribution in https://github.com/dbt-msft/dbt-sqlserver/pull/194
* @pre-commit-ci made their first contribution in https://github.com/dbt-msft/dbt-sqlserver/pull/232

**Full Changelog**: https://github.com/dbt-msft/dbt-sqlserver/compare/v1.0.0...v1.1.0.rc1

<hr /><em>This discussion was created from the release <a href='https://github.com/dbt-msft/dbt-sqlserver/releases/tag/v1.1.0.rc1'>v1.1.0.rc1</a>.</em>

**Comment by JamesLiSooeyUiPath** [comment]
Getting the below issue after running command "pip install --upgrade dbt-snowflake". Is dbt-sqlserver still incompatible with dbt-core 1.1.0?

dbt-sqlserver 1.0.0 requires dbt-core~=1.0.0, but you have dbt-core 1.1.0 which is incompatible.

**Reply by sdebruyn** [reply]
Could you try with `pip install dbt-sqlserver==1.1.0rc1`?
